integrations/pytorch/tutorials/classification_sparse_transfer_learning_tutorial.md

```diff
@@ -114,7 +115,18 @@ the `checkpoint-path` can be set to `zoo` for auto-downloading corresponding che
 To learn more about the usage, run script with `-h` option to see help. 
 
 The script automatically saves model checkpoints after each epoch and reports the validation loss along with layer sparsities. 
-The model is saved in [ONNX](https://onnx.ai/) format and can be loaded later for inference or other experiments.
+
+### ONNX Export
+The model can be saved to [ONNX](https://onnx.ai/) format to be loaded later for inference or other experiments using
+the `sparseml.image_classification.export` script.
```

Comment:
Noting it may be confusing to the user that we refer to it as the `sparseml.image_classification.export` script when the command is sparseml.image_classification.export_onnx